packages/cli/src/bin/utils/app-api.ts

```diff
@@ -0,0 +1,200 @@
+import { readFileSync } from 'node:fs';
+import { Spinner } from './spinner.js';
+import { chalk } from './format.js';
+import { AppConfig } from '../../lib/app-config.js';
+
+export type FusionEnv = 'ci' | 'fqa' | 'fprd';
+
+/* build api endpoint url */
+export const getEndpointUrl = (
+    endpoint: string,
+    fusionEnv: FusionEnv,
+    version: string = '1.0-preview',
+) => {
+    const { FUSION_APP_API: appApiUrl } = process.env;
+    return (
+        appApiUrl ??
+        `https://fusion-s-apps-${fusionEnv}.azurewebsites.net/${endpoint}?api-version=${version}`
+    );
```

Comment:
i think we should resolve the app service from service discovery, but `preview` might not be there yet

Reply:
Done